tests/Unit/PointwiseFunctions/AnalyticSolutions/Elasticity/HalfSpaceMirror.py

```diff
@@ -8,133 +8,120 @@
-
-    xi_z = integrate.quad(
-        lambda k: integrand_xi_z(k, w, z, local_lame_parameter, shear_modulus,
-                                 beam_width, applied_force),
-        0,
-        np.inf,
-        limit=100,
-        epsabs=1e-13)[0]
-
-    if w == 0.:
-        assert xi_w <= 1e-13, 'xi_w is not zero at the origin'
-        cos_phi = 0.
-        sin_phi = 0.
-    else:
-        cos_phi = x / float(w)
-        sin_phi = y / float(w)
+    radius = np.sqrt(x**2 + y**2)
+    if radius <= 1e-13:
+        return np.zeros(3)
+    xi_w = integrate.quad(lambda k: integrand_xi_w(
+        k, radius, z, local_lame_parameter, shear_modulus, beam_width),
+                          0,
+                          np.inf,
+                          limit=100,
+                          epsabs=1e-13)[0]
+
+    xi_z = integrate.quad(lambda k: integrand_xi_z(
+        k, radius, z, local_lame_parameter, shear_modulus, beam_width),
+                          0,
+                          np.inf,
+                          limit=100,
+                          epsabs=1e-13)[0]
+
+    cos_phi = x / float(radius)
+    sin_phi = y / float(radius)
     return np.asarray([xi_w * cos_phi, xi_w * sin_phi, xi_z])
 
 
-def strain(r, beam_width, applied_force, bulk_modulus, shear_modulus):
+def strain(r, beam_width, bulk_modulus, shear_modulus):
     local_lame_parameter = lame_parameter(bulk_modulus, shear_modulus)
 
     x = r[0]
     y = r[1]
     z = r[2]
-    w = np.sqrt(x**2 + y**2)
+    radius = np.sqrt(x**2 + y**2)
 
-    theta = integrate.quad(
-        lambda k: integrand_theta(k, w, z, local_lame_parameter, shear_modulus,
-                                  beam_width, applied_force),
-        0,
-        np.inf,
-        limit=100,
-        epsabs=1e-13)[0]
+    theta = integrate.quad(lambda k: integrand_theta(
+        k, radius, z, local_lame_parameter, shear_modulus, beam_width),
+                           0,
+                           np.inf,
+                           limit=100,
+                           epsabs=1e-13)[0]
 
     strain_rz = integrate.quad(lambda k: integrand_strain_rz(
-        k, w, z, local_lame_parameter, shear_modulus, beam_width, applied_force
-    ),
+        k, radius, z, local_lame_parameter, shear_modulus, beam_width),
                                0,
                                np.inf,
                                limit=100,
                                epsabs=1e-13)[0]
 
     strain_zz = integrate.quad(lambda k: integrand_strain_zz(
-        k, w, z, local_lame_parameter, shear_modulus, beam_width, applied_force
-    ),
+        k, radius, z, local_lame_parameter, shear_modulus, beam_width),
                                0,
                                np.inf,
                                limit=100,
                                epsabs=1e-13)[0]
 
-    if w == 0:
-        w = 1.
+    if radius == 0:
+        radius = 1.
         strain_rr = 0.5 * (theta - strain_zz)
         strain_pp = strain_rr
```

Comment:
return the strain at the origin directly instead of "faking" the transform to cartesian coords